### ISSUE TITLE
Uncaught TypeError: Path must be a string. Received undefined

### ISSUE THREAD

**Issue by krusnof** [sindresorhus/atom-editorconfig#67]
1. Install via terminal
2. Launch Editorconfig via Command Palette
3. It fails with below info

**Atom Version**: 1.5.4
**System**: Mac OS X 10.11.3
**Thrown From**: [editorconfig](https://github.com/sindresorhus/atom-editorconfig) package, v1.2.4

### Stack Trace

Uncaught TypeError: Path must be a string. Received undefined

```
At path.js:8

TypeError: Path must be a string. Received undefined
    at assertPath (path.js:8:11)
    at Object.posix.join (path.js:477:5)
    at atom-workspace.init (/Users/Kruse/.atom/packages/editorconfig/commands/generate.js:6:26)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:260:29)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61
    at CommandPaletteView.module.exports.CommandPaletteView.confirmed (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/command-palette/lib/command-palette-view.js:183:32)
    at CommandPaletteView.module.exports.SelectListView.confirmSelection (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-space-pen-views/lib/select-list-view.js:338:21)
    at space-pen-div.atom.commands.add.core:confirm (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-space-pen-views/lib/select-list-view.js:109:19)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:260:29)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:570:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:378:22)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeydown (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:97:36)
    at HTMLDocument.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:3:61)
```

### Commands

```
     -1:00.5.0 settings-view:open (atom-text-editor.editor.is-focused)
     -0:04.5.0 command-palette:toggle (div.package-detail.panels-item)
     -0:00.8.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:00.8.0 EditorConfig:generate-config (div.package-detail.panels-item)
```

### Config

```json
{
  "core": {}
}
```

### Installed Packages

```coffee
# User
color-picker, v2.1.1
editorconfig, v1.2.4
file-icons, v1.6.18
floobits, v0.31.0
git-log, v0.4.1
highlight-line, v0.11.1
minimap, v4.20.0
minimap-git-diff, v4.2.0

# Dev
No dev packages
```

**Comment by sindresorhus** [comment]
It assumed `atom.project.getPaths()` always returns something, which it seems it doesn't. Not exactly sure how to handle this. Suggestions welcome. https://github.com/sindresorhus/atom-editorconfig/blob/4cb3675cdda44106b0382f7013a5ecb7041912ec/commands/generate.js#L6

**Comment by florianb** [comment]
I guess the error is caused by the fact you're using the first path without checking if the returned Array is empty. I would submit a patch, but since the project is untested i'm afraid breaking things.

**Comment by Arcanemagus** [comment]
Btw, a file can be opened in Atom _without_ being part of a project, which is how you would get into that situation.

**Comment by florianb** [comment]
This will be fixed with the PR #82
